aptos-move/move-examples/drand/sources/lottery.move

```diff
@@ -129,7 +143,7 @@ module drand::lottery {
     /// Allows anyone to close the lottery (if enough time has elapsed) and to decide the winner, by uploading
     /// the correct _drand-signed bytes_ associated with the committed draw time in `Lottery::draw_at`.
     /// These bytes will then be verified and used to extract randomness.
-    public entry fun close_lottery(drand_signed_bytes: vector<u8>): Option<address> acquires Lottery {
+    public entry fun close_lottery(drand_signed_bytes: vector<u8>) acquires Lottery {
```

Comment:
Yep... Ideally, if you can, consider having the `public entry close_lottery(...)` function call a `public close_lottery_internal(...):  Option<address>` function, where everything is implemented, and ignore its return value.

This would allow other contracts to call `drand::lottery::close_lottery_internal` and check who won. Right now, there is no easy way of doing this.